main.js

```diff
@@ -19,15 +19,19 @@ function Hoverable() {
 	var win = window;
 
 	function init() {
-		win.document.body.setAttribute('data-o-hoverable--js', '');
+		win.document.documentElement.setAttribute('data-o-hoverable--js', '');
 		touchSupport();
 	}
 
+	function updateClasses(action) {
```

Comment:
> I use it on documentElement

In that case it won't be working on your sites at present! 

I'm with Alberto - a minor release avoids dependency update hell, and the extra lines of code/processing to provide backwards compatibility are pretty teeny.

I'm not sure I understand the point about demos, but because of the use of `win.doc` here https://github.com/Financial-Times/o-hoverable/blob/master/main.js#L29 maybe they're all broken anyway. I have heard anecdotal evidence that o-hoverable is failing to detect touch on devices at the moment.